server/schemas/mutations/editJob.js

```diff
@@ -0,0 +1,9 @@
+const Job = require('../../models');
+
+async function editJob(parent, { updateJob}) {
+    const job = await Job.findOneAndUpdate(updateJob);
```

Comment:
To perform the findOne operation we need a filter as the first input passed into the function, we can use the jobId for this. Then to make sure we get back the updated document we want to include along the new:true option. 

```suggestion
    const job = await Job.findOneAndUpdate({_id: jobId},updateJob,{new: true});
```

same change for editProposal